### PR TITLE
Changelog - 9.5.0 & removed deprecated modules

### PR DESCRIPTION
**Pull request by Saksham-Nautiyal**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by sachin-apa — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
 
+v9.5.0
+======
+
+Release Summary
+---------------
+
+- The ``idrac_redfish_storage_controller`` module is enhanced to support secure and/or cryptographic erase of the physical disk.
+- The ``ome_application_network_proxy`` module is enhanced to allow you to manage the Proxy Exclusion List in the Proxy Configuration.
+- The ``idrac_redfish_powerstate`` module is enhanced to add support for full virtual A/C power cycle in server powercycle module.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- The ``idrac_redfish_powerstate`` module is enhanced to support full virtual A/C power cycle.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
+
+Minor Changes
+-------------
+
+- idrac_redfish_powerstate - This module is enhanced to add support for full virtual A/C power cycle option in server powercycle module.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_redfish_powerstate - This module is enhanced to support enhanced to support full virtual A/C power cycle.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
+- idrac_redfish_storage_controller - This module is enhanced to allow you to add support for secure and/or cryptographic erase of the physical disk.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- idrac_redfish_storage_controller - This module is enhanced to support secure and/or cryptographic erase of the physical disk.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
+- ome_application_network_proxy - This module is enhanced to allow you to manage the Proxy Exclusion List in the Proxy Configuration.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- ome_application_network_proxy - This module is enhanced to manage the Proxy Exclusion List and Certificate Validation. 
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
 .. contents:: Topics
 
+v9.5.0
+======
+
+Release Summary
+---------------
+
+- The ``idrac_redfish_storage_controller`` module is enhanced to support secure and/or cryptographic erase of the physical disk.
+- The ``ome_application_network_proxy`` module is enhanced to allow you to manage the Proxy Exclusion List in the Proxy Configuration.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- The ``ome_application_network_proxy`` module is enhanced to manage the Proxy Exclusion List and Certificate Validation
```

File: README.md [hunk 6]
```diff
@@ -82,7 +82,7 @@ For more information about how to use the collection, refer to [OME modules](htt
 
 For more information about Unit testing, see [Unit testing](https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/tests/README.md).
 
-For more information about Integration testing, see [Integration testing](https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/tests/integrations/README.md).
+For more information about Integration testing, see [Integration testing](https://github.com/dell/dellemc-openmanage-ansible-modules/tree/collections/tests/integration).
```
Comment (rajshekarp87) [inline README.md]:
```suggestion
For more information about Integration testing, see [Integration testing](https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/tests/integration/README.md).
```

File: CHANGELOG.rst [hunk 7]
```diff
@@ -4,6 +4,32 @@ Dell OpenManage Ansible Modules Release Notes
...
 
+v9.5.0
+======
+
+Release Summary
+---------------
+
+- The ``idrac_redfish_storage_controller`` module is enhanced to support secure and/or cryptographic erase of the physical disk.
+- The ``ome_application_network_proxy`` module is enhanced to manage the Proxy Exclusion List and Certificate Validation.
+- The ``idrac_redfish_powerstate`` module is enhanced to support for full virtual A/C power cycle.
```
Comment (singh-lovepreet1) [inline CHANGELOG.rst]:
```suggestion
- The ``idrac_redfish_powerstate`` module is enhanced to support full virtual A/C power cycle.
```

**Review by rajshekarp87 — APPROVED** [review]

**Review by ShivamSh3 — APPROVED** [review]

**Pull request merged**